README.md

```diff
@@ -131,6 +131,7 @@ a great responsiblity to never betray this trust:
 * 🔻 Apply the [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege).
 * 🤐 [Never commit secrets, keys or passwords](https://mobify.atlassian.net/wiki/display/SYS/Best+Practices+For+Keeping+Secrets+in+Applications).
 * ⏰ [Patch regularly](https://docs.google.com/document/d/1J3oC-7VJYC-wDLIF04M1jN79IhjF3sBLap0TUBJDVaw/edit#).
+* 💳 Be aware of our [PCI programme](https://mobify.atlassian.net/wiki/spaces/PCI/) and how it affects your work.
```

Comment:
Would be nice to define the acronym here. The emoji gives a good hint though!

Reply:
Done - thanks.